flake.nix

```
@@ -2,7 +2,7 @@
   description = "NixOS WSL";
 
   inputs = {
-    nixpkgs.url = "nixpkgs/nixos-22.05";
+    nixpkgs.url = "nixpkgs/nixos-22.11";
```

```suggestion
    nixpkgs.url = "github:NixOS/nixpkgs/nixos-22.11";
```
otherwise the lock file will be incorrect if people overwrite the registry entry